ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -8,6 +8,14 @@ import UIKit
 class TabBarItemView: UIView {
     let item: TabBarItem
 
+    var isEnabled: Bool = true {
+        didSet {
+            titleLabel.isEnabled = isEnabled
+            imageView.tintAdjustmentMode = isEnabled ? .automatic : .dimmed
+            isUserInteractionEnabled = isEnabled
```

Comment:
Having to call isUserInteractionEnabled is usually a sign we're doing weird things that UIButtons and related types should be doing for free for us, but I guess that's pre-existing for TabBarItemView since it inherits from UIView?

Reply:
could just disable the gesture recognizer? Otherwise won't something like large content viewer also not work? Actually, should that work on disabled items?